### PR TITLE
Preliminary batch script for running bin2tif via PBS

### PR DESCRIPTION
**Pull request by craig-willis**
* Updated terra_bin2tif.py to use new temp directory approach 
* Added bin2tif sample PBS script 
* Updated readme with steps to reproduce

See also:
https://github.com/terraref/computing-pipeline/issues/368

### CODE REVIEW

File: bin2tif/readme.md [hunk 1]
```diff
@@ -42,6 +42,69 @@ docker run \
...
+module load pythonlibs/2.7.10
+virtualenv terraref --python=python2.7
+cd terraref
+. bin/activate
+git clone https://opensource.ncsa.illinois.edu/stash/scm/cats/pyclowder2.git
+pip install  pyclowder2/
+git clone https://github.com/terraref/terrautils/
+pip install terrautils/
+git clone https://github.com/terraref/extractors-stereo-rgb
+pip install Pillow enum34 pyyaml pika functools32 pyparsing pytz GDAL
```
Comment (max-zilla) [inline bin2tif/readme.md]: we should also be able to do pip install -r requirements.txt to install these dependencies for terrautils/pyclowder2, that will at least cover Pillow, pika, enum, etc.

File: bin2tif/readme.md [hunk 2]
```diff
@@ -42,6 +42,69 @@ docker run \
...
+export CLOWDER_USER=
+export CLOWDER_PASS=
+export CLOWDER_SPACE
+export INFLUXDB_PASSWORD=
+```
+
+Run the job:
+```
+. env.sh
+qsub -V bin2tif.pbs
```
Comment (max-zilla) [inline bin2tif/readme.md]: can we just include ```. env.sh``` as first line in the pbs script?

File: bin2tif/readme.md (hunk 1 above)
Comment (craig-willis) [reply bin2tif/readme.md]: Actually, your existing environment in /projects/arpae/terraref/shared/extractors makes more sense to me than how I've done it.

File: bin2tif/readme.md (hunk 2 above)
Comment (craig-willis) [reply bin2tif/readme.md]: Yes, and we could just put that env file in /projects/arpae/terraref/shared/extractors

Comment (craig-willis) [reply bin2tif/readme.md]: (I didn't realize you had already created these batch processes ... seems like I've wasted some time.)

Comment (max-zilla) [reply bin2tif/readme.md]: mine were very basic, didnt support dynamic changing of environment vars or the parallel library. i think those are good improvements

**Pull request merged**